frontend/components/Header/Nav/index.js

```diff
@@ -17,25 +16,28 @@ import SubNav from './SubNav';
 
 import type { NavType } from './__config__';
 
-const NavStyled = styled('nav')(
-    {
-        [tablet]: {
-            maxWidth: '740px',
-        },
-        [desktop]: {
-            maxWidth: '980px',
-        },
-        [leftCol]: {
-            maxWidth: '1140px',
-        },
-        [wide]: {
-            maxWidth: '1300px',
-        },
-        position: 'relative',
-        margin: '0 auto',
-    },
-    clearFix,
-);
+const centered = css`
+    ${tablet} {
+        max-width: 740px;
+    }
+    ${desktop} {
+        max-width: 980px;
+    }
+    ${leftCol} {
+        max-width: 1140px;
+    }
+    ${wide} {
+        max-width: 1300px;
+    }
```

Comment:
We might want to eventually pull this style out or have a pattern for mapping over all breakpoints. (But not today)